spring-cloud-kubernetes-fabric8-autoconfig/src/test/java/org/springframework/cloud/kubernetes/Fabric8PodUtilsTest.java

```diff
@@ -38,7 +38,7 @@
 import org.springframework.cloud.kubernetes.fabric8.Fabric8PodUtils;
 
 @SuppressWarnings("unchecked")
-public class Fabric8PodUtilsTest {
```

Comment:
removed `public` everywhere from tests, since I was changing some classes in this package anyway